examples/component/http-server/README.md

```diff
@@ -0,0 +1,179 @@
+# TinyGo HTTP Password Checker
+
+This repository contains a WebAssembly Component written in [TinyGo][tinygo], which:
```

Comment:
Let's swap `TinyGo` for `Go` here as well, can include the same language about `(compiled to WebAssembly using [TinyGo][tinygo])` bit if it makes sense.